rfcs/0173-hotline.md

```diff
@@ -0,0 +1,234 @@
+
+```
+Willkommen bei dem Ziehbegehren 299106.
+Bitte drücken Sie die 1, um alle Kommentare vorzulesen.
+Drücken Sie die 2, um dieses Ziehbegehren zu schliessen.
+Drücken Sie die 3, um dieses Ziehbegehren zu verschmelzen.
+Drücken Sie die 4, um eine Rezension zu hinterlassen.
+```
+
+- User presses 3
+
+```
+Bitte geben Sie Ihren Privatschlüssel jetzt im Nummernfeld an. Achten Sie bitte darauf, dass Sie Ihren Schlüssel erst in Basis 10 konvertiert haben müssen.
+```
+
+- User presses 111 112 101 110 115 115 104 45 107 101 121 45 118 49 0 0 0 0 4 110 111 110 101 0 0 0 4 110 111 110 101 0 0 0 0 0 0 0 1 0 0 0 51 0 0 0 11 115 115 104 45 101 100 50 53 53 49 57 0 0 0 32 81 15 176 105 61 9 26 72 71 82 53 65 127 33532 105 49882 26415 38 114 96 1859 38 7 896 0 0 1014 80 41974 80 40960 0 11 115 115 104 45 101 100 50 53 53 49 57 0 0 0 32 81 15 176 105 61 9 26 72 71 82 53 65 127 33532 105 49882 26415 38 114 96 1859 38 7 896 0 0 64 2399 1243 115 101 112 89 27 34727 33 107 6827 64594 80 86 80 125 43 10 36 49 9295 176 105 61 9 26 72 71 82 53 65 127 33532 105 49882 26415 38 114 96 1859 38 7 896 0 0 12 108 97 115 115 64 105 103 110 97 118 105 97 2
+
+```
+Leider ist der von Ihnen eingegebene Schlüssel nicht korrekt, versuchen Sie es bitte erneut.
+```
+
+- User presses 111 112 101 110 115 115 104 45 107 101 121 45 118 49 0 0 0 0 4 110 111 110 101 0 0 0 4 110 111 110 101 0 0 0 0 0 0 0 1 0 0 0 51 0 0 0 11 115 115 104 45 101 100 50 53 53 49 57 0 0 0 32 81 15 176 105 61 9 26 72 71 82 53 65 127 33532 105 49882 26415 38 114 96 1859 38 7 896 0 0 1014 80 41974 80 40960 0 11 115 115 104 45 101 100 50 53 53 49 57 0 0 0 32 81 15 176 105 61 9 26 72 71 82 53 65 127 33532 105 49882 26415 38 114 96 1859 38 7 896 0 0 64 2399 1243 115 101 112 89 27 34727 33 107 6827 64594 80 86 80 125 43 10 36 49 9295 176 105 61 9 26 72 71 82 53 65 127 33532 105 49882 26415 38 114 96 1859 38 7 896 0 0 12 108 97 115 115 64 105 103 110 97 118 105 97 1
+
+```
+Vielen Dank, das Ziehbegehren 299106 wurde nun in den Hauptzweig verschmolzen. Auf Wiederhören.
+```
+
+## translation of example interaction
+
+- User dials `+49 0800 64967(NIXOS)`
+- AI voice greets caller in German:
+
+```
+Welcome to the Github pull request assistant for Nixpkgs.
+Please press 1 to get the list of open pull requests.
+Press 2 if you want to go to a specific pull request.
+Press 3 if you want to filter after a specific user.
+```
+- User presses 3
+
+```
+Please tell us the name of the user you want to filter.
+```
+- User says Mic92
+
+```
+I understood MichaelTwohundred, is this correct?
+```
+
+- User says no
+
+
+```
+Could you please repeat the name?
+```
+
+- User says Mic92
+
+```
+I understood MicNinetyThree, is this correct?
+```
+
+- User says no
+```
+Could you please repeat the name?
+```
+
+- User says Mic92
+
+```
+I understood MicNineThree, is this correct?
+```
+
+- User says no
+
+```
+Sadly all our speech to text pipelines are busy. Instead we will read to you all pull requests by r-ryantm.
```

Comment:
Before listing all PRs I think we can help the user by giving an indication of the number of PRs that are going to be listed:
```suggestion
Sadly all our speech to text pipelines are busy. Instead we will read to you all 515 open pull requests by r-ryantm.
```